cainday_bradley/style.css

```
@@ -0,0 +1,234 @@
+h1 {
+    text-align: center;
+    border: 2px solid black;
+    background-color: #ffeff2;
+}
+
+body {
+    background-color: #ffdae0;
+}
+
+h2 {
+    background-color: #ffeff2;
+    border: 2px solid black;
+    padding: 10px; 
+    border-radius: 20%;     
+}
+
+h3 {
+    background-color: #ffeff2;
+    border: 2px solid black;
+    padding: 8px; 
+    border-radius: 20%;     
+}
+
+.interests-section {
+    text-align: center;
+    border: 2px solid black;
+    padding: 40px;
+    background-color: #ffc0cb;
+}
+
+.interests-section ul {
+    list-style-position: inside; 
+    padding-left: 0;
+    list-style-type: disc;
+    margin-left: 20px;
+    margin-bottom: 60px;
+}
+
+.interests-section li {
+    margin: 0;
+    font-size: 18px;
+    margin-bottom: 10px;
+}
+
+.interests-section h2 {
+    font-size: 24px;
+    margin-top: 20px;
+}
+
+.works-section {
+    text-align: center;
+    border: 2px solid black;
+    padding: 40px;
+    background-color: #ffa7b6;
+}
+
+.content {
+    max-width: 1500px;
+    margin: 0 auto;
+    padding: 20px;
+    display: flex; 
+    justify-content: space-between; 
+}
+
+.column {
+    width: 48%; 
+    padding: 20px;
+}
+
+.works-section ul {
+    list-style: none;
+    padding-left: 0;
+}
+
+.works-section a {
+    font-size: 18px;
+    text-decoration: none;
+    color: #333;
+    font-weight: bold;
+    margin-right: 10px;
+    line-height: 4;
+}
+
+.column {
+    margin-bottom: 40px;
+}
+
+.testimonials-section {
+    margin: 0 auto;
+    max-width: 1500px;
+}
+
+.gallery-section {
+    margin: 0 auto;
+    max-width: 1500px;
+}
+
+.testimonials-section {
+  font-size: 17px;
+  text-align: center;
+  border: 2px solid black;
+  padding: 20px;
+  background-color: #ff8da1;
+  margin-bottom: 40px; 
+  width: 48%; 
+  line-height: 1.5;
+}
+
+.gallery-section {
+  font-size: 17px;
+  text-align: center;
+  border: 2px solid black;
+  padding: 20px;
+  background-color: #ff748c;
+  margin-bottom: 40px; 
+  width: 48%; 
+  line-height: 1.5;
+}
+
+
+.gallery-section img {
+  border: 2px solid black;
+  background-color: #fae2e3;
+  border-radius: 20%;
+  width: 130px; 
+}
+
+.goals {
+    border: 2px solid black;
+    display: grid;
+    grid-template-areas:
+    "itm1 itm2";
+}
+
+
+h4 {
```

```suggestion

h4 {
```
Remove space. CSS block should be seperated by a single space only.